bn254/src/lib.rs

```diff
@@ -152,6 +156,15 @@ mod target_arch {
             reverse_copy(&be_bytes[FIELD_SIZE..], &mut pod_bytes[FIELD_SIZE..])?;
             Ok(Self(pod_bytes))
         }
+
+        #[inline(always)]
+        fn from_le_bytes(le_bytes: &[u8]) -> Result<Self, AltBn128Error> {
```

Comment:
I can't comment on non-diff lines, but can you update the docs on top of `pub struct PodG1(pub [u8; G1_POINT_SIZE]);` especially regarding
```
/// The Solana syscalls still expect the inputs to be encoded in big-endian as
/// specified in EIP-197. The type `PodG1` is an intermediate type that
/// facilitates the translation between the EIP-197 encoding and the arkworks
/// implementation encoding.
```